### PR TITLE
Update content on landing pages.

### PR DESCRIPTION
**Pull request by SwordSaintLancelot**
### Motivation
As per the requirements a few editorial changes needed to be made on the landing pages of JupyterHub, API information and US GHG Center.

### Changes
The edits were made in the qmd scripts, namely jupyterhub.qmd, apis.qmd and index.qmd

### CODE REVIEW

File: index.qmd [hunk 1]
```diff
@@ -5,19 +5,19 @@ subtitle: "Resources for the US GHG Center"
...
-4. Data **processing and verification** reports, e.g. for the [CEOS CH₄ budget yearly](processing_and_verification_reports/gosat-based-ch4budget-yeargrid-v1_Processing and Verification Report.qmd) dataset.
+1. Services provided for accessing and analyzing the US GHG Center datasets, such as the [JupyterHub](services/jupyterhub.qmd) environment for interactive computing.
+2. Dataset usage examples, e.g. for the [Wetland Methane Emissions from the LPJ-wsl model](user_data_notebooks/lpjwsl-wetlandch4-grid-v1_User_Notebook.ipynb) dataset, that shows how to load the dataset in Python in [JupyterHub](services/jupyterhub.qmd).
+3. Dataset transformation scripts, such as the [CASA-GFED3 Land Carbon Flux](cog_transformation/casagfed-carbonflux-monthgrid-v3.ipynb) dataset.
+4. Data processing and verification reports that openly present the process we used to check and verify that any transformation did not alter the original source data. An example is the [CEOS CH₄ budget yearly](processing_and_verification_reports/gosat-based-ch4budget-yeargrid-v1_Processing and Verification Report.qmd) dataset.
```
Comment (slesaad) [inline index.qmd:18]: can you keep the bold words like before?

File: index.qmd [hunk 2]
```diff
@@ -5,19 +5,19 @@ subtitle: "Resources for the US GHG Center"
...
 
 
 ## Contact
 
-For technical and usage questions, please contact us at [support@ghg.center](mailto:support@ghg.center) or via the Feedback forms at [earth.gov/ghgcenter](https://earth.gov/ghgcenter).
\ No newline at end of file
+For technical help or general questions, please contact the support team using the [feedback form](https://docs.google.com/forms/d/e/1FAIpQLSeVWCrnca08Gt_qoWYjTo6gnj1BEGL4NCUC9VEiQnXA02gzVQ/viewform)
```
Comment (slesaad) [inline index.qmd:23]: please add a period at . the end

File: services/jupyterhub.qmd [hunk 3]
```diff
@@ -2,24 +2,27 @@
...
+If you are a scientist affiliated with other NASA projects such as [VEDA](https://nasa-impact.github.io/veda-docs/), [EIS](https://eis.smce.nasa.gov/about.html), and [MAAP](https://ops.maap-project.org), you can also keep using the resources provided by these projects. Through the use of open-source technology, we make sure our services are interoperable and exchangeable.
 
 
-## Getting access to the GHG Center JupyterHub environment
+## To Get US GHG Center JupyterHub access:
 
-Access to the GHG Center notebook environment is currently on an as-need basis. If you are a user affiliated with the GHG Center, you can gain access by following these steps:
+ The US GHG Center notebook environment is available to authorized users on an as-need basis. If you are a user affiliated with the US GHG Center, you can gain access by using our Hub Access Request form.following these steps:
```
Comment (slesaad) [inline services/jupyterhub.qmd:14]:
```suggestion
 The US GHG Center notebook environment is available to authorized users on an as-need basis. If you are a user affiliated with the US GHG Center, you can gain access by using our [Hub Access Request form](https://forms.gle/4Z7rkUkeK9MW4joi7).
```

File: services/jupyterhub.qmd [hunk 4]
```diff
@@ -2,24 +2,27 @@
...
+* Make sure you have a [Github Account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account). Take note of your [Github username](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/remembering-your-github-username-or-email).
+* Fill out the request form and provide needed information.
+* Watch your email for notification of authorization and the invite to join the [US GHG Center Hub Access Github Team](https://github.com/orgs/US-GHG-Center/teams/ghgc-hub-access).
 
-* Send an email to the GHG Center team ([hub@ghg.center](mailto:hub@ghg.center)) asking for access to the GHG Center notebook environment. Please include your Github username. They will invite you through Github to join the [GHG Center Hub Access Github Team](https://github.com/orgs/US-GHG-Center/teams/ghgc-hub-access). Please watch your email for the invite.
+* Once you accept the invitation, you can go to [hub.ghg.center](https://hub.ghg.center/) and login using your Github credentials.
```
Comment (slesaad) [inline services/jupyterhub.qmd:20]:
```suggestion
* Once you accept the invitation, you can go to [hub.ghg.center](https://hub.ghg.center/) and login using your GitHub credentials.
```

Comment (slesaad) [reply services/jupyterhub.qmd:20]: replace all Github -> GitHub

File: index.qmd (hunk 1 above)
Comment (SwordSaintLancelot) [reply index.qmd:18]: It seemed different from the content that deborah provided that's why I changed it.

File: services/jupyterhub.qmd [hunk 5]
```diff
@@ -2,24 +2,27 @@
...
-* Once you accepted the invitation, you should be able to go to [hub.ghg.center](https://hub.ghg.center/) and login via your Github credentials.
 
+## To access User Notebooks
 
-## Instructory notebooks
+This site provides [Jupyter notebooks](../user_data_notebooks/casagfed-carbonflux-monthgrid-v3_User_Notebook.ipynb) showing how to load and analyze Earth data in the interactive cloud computing environment.
 
-This documentation site provides [Jupyter notebooks](../user_data_notebooks/casagfed-carbonflux-monthgrid-v3_User_Notebook.ipynb) on how to load and analyze Earth data an interactive cloud computing environment.
\ No newline at end of file
+Further instructions are included in each notebook.
```
Comment (slesaad) [inline services/jupyterhub.qmd:27]:
```suggestion
Further instructions are included in each notebook. 

```

File: services/jupyterhub.qmd [hunk 6]
```diff
@@ -2,24 +2,27 @@
...
 
-Access to the GHG Center notebook environment is currently on an as-need basis. If you are a user affiliated with the GHG Center, you can gain access by following these steps:
+ The US GHG Center notebook environment is available to authorized users on an as-need basis. If you are a user affiliated with the US GHG Center, you can gain access by using our Hub Access Request form.following these steps:
 
-* Make sure you have a [Github Account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account). Take note of your [Github username](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/remembering-your-github-username-or-email)
+* Make sure you have a [Github Account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account). Take note of your [Github username](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/remembering-your-github-username-or-email).
+* Fill out the request form and provide needed information.
```
Comment (slesaad) [inline services/jupyterhub.qmd:17]:
```suggestion
* Fill out the [request form](https://forms.gle/4Z7rkUkeK9MW4joi7) and provide needed information.
```

File: services/jupyterhub.qmd [hunk 7]
```diff
@@ -6,23 +6,23 @@ The US GHG Center promotes the use of JupyterHub environments for interactive da
...
+If you are a scientist affiliated with other NASA projects such as [VEDA](https://nasa-impact.GitHub.io/veda-docs/), [EIS](https://eis.smce.nasa.gov/about.html), and [MAAP](https://ops.maap-project.org), you can also keep using the resources provided by these projects. Through the use of open-source technology, we make sure our services are interoperable and exchangeable.
 
 
 ## To Get US GHG Center JupyterHub access:
 
- The US GHG Center notebook environment is available to authorized users on an as-need basis. If you are a user affiliated with the US GHG Center, you can gain access by using our Hub Access Request form.following these steps:
+ The US GHG Center notebook environment is available to authorized users on an as-need basis. If you are a user affiliated with the US GHG Center, you can gain access by using our [Hub Access Request form](https://forms.gle/4Z7rkUkeK9MW4joi7)
```
Comment (slesaad) [inline services/jupyterhub.qmd:14]: missing a period in the end

File: services/jupyterhub.qmd [hunk 8]
```diff
@@ -6,23 +6,23 @@ The US GHG Center promotes the use of JupyterHub environments for interactive da
 
 Users affiliated with the US GHG Center can get access to a dedicated JupyterHub service, provided in collaboration with [2i2c](https://2i2c.org/): [hub.ghg.center](https://hub.ghg.center/). Please find instructions for requesting access below.
 
-If you are a scientist affiliated with other NASA projects such as [VEDA](https://nasa-impact.github.io/veda-docs/), [EIS](https://eis.smce.nasa.gov/about.html), and [MAAP](https://ops.maap-project.org), you can also keep using the resources provided by these projects. Through the use of open-source technology, we make sure our services are interoperable and exchangeable.
+If you are a scientist affiliated with other NASA projects such as [VEDA](https://nasa-impact.GitHub.io/veda-docs/), [EIS](https://eis.smce.nasa.gov/about.html), and [MAAP](https://ops.maap-project.org), you can also keep using the resources provided by these projects. Through the use of open-source technology, we make sure our services are interoperable and exchangeable.
```
Comment (slesaad) [inline services/jupyterhub.qmd:9]: revert back to lowercase github in urls

Comment (slesaad) [reply services/jupyterhub.qmd:9]: in all the urls

File: services/jupyterhub.qmd [hunk 9]
```diff
@@ -6,23 +6,23 @@ The US GHG Center promotes the use of JupyterHub environments for interactive da
...
-* Once you accept the invitation, you can go to [hub.ghg.center](https://hub.ghg.center/) and login using your Github credentials.
+* Once you accept the invitation, you can go to [hub.ghg.center](https://hub.ghg.center/) and login using your GitHub credentials.
 
 
 ## To access User Notebooks
 
 This site provides [Jupyter notebooks](../user_data_notebooks/casagfed-carbonflux-monthgrid-v3_User_Notebook.ipynb) showing how to load and analyze Earth data in the interactive cloud computing environment.
 
-Further instructions are included in each notebook.  
+Further instructions are included in each notebook.
```
Comment (slesaad) [inline services/jupyterhub.qmd:27]: add a line break after "notebook."

this all would have been super easy if you had just accepted my suggestions directly

File: services/jupyterhub.qmd [hunk 10]
```diff
@@ -11,7 +11,7 @@ If you are a scientist affiliated with other NASA projects such as [VEDA](https:
 
 ## To Get US GHG Center JupyterHub access:
 
- The US GHG Center notebook environment is available to authorized users on an as-need basis. If you are a user affiliated with the US GHG Center, you can gain access by using our [Hub Access Request form](https://forms.gle/4Z7rkUkeK9MW4joi7).
+ The US GHG Center notebook environment is available to authorized users on an as-need basis. If you are a user affiliated with the US GHG Center, you can gain access by using our [Hub Access Request form](https://docs.google.com/forms/d/e/1FAIpQLSdai8otCdrVQzJgev8mjDhzKyCg7jcrB3UeTXNHoCiaMKrkaQ/viewform).
 
 * Make sure you have a [GitHub Account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account). Take note of your [GitHub username](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/remembering-your-github-username-or-email).
 * Fill out the [request form](https://forms.gle/4Z7rkUkeK9MW4joi7) and provide needed information.
```
Comment (slesaad) [inline services/jupyterhub.qmd:17]: the url needs to be updated here too

**Review by slesaad — APPROVED** [review]

**Pull request merged**